index.html

```diff
@@ -27,14 +24,138 @@ <h1>
       I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
       </p>
       <h3>LETS CONNECT</h3>
-      <ul>
+      <ul class="socials">
         <li><a href="#"><img src="images/Tweeter.png"></a></li>
         <li><a href="#"><img src="images/Linkedin icon.png"></a></li>
         <li><a href="#"><img src="images/m.png"></a></li>
         <li><a href="#"><img src="images/Github.png"></a></li>
         <li><a href="#"><img src="images/tech.png"></a></li>
       </ul>
     </section>
+  </header>
+  <main class="main">
+    <section id="first-work">
+      <div class="snap">
+        <img src="images/Snapshoot Portfolio.png" width="311px" height="220px" class="snap">
+      </div>
```

Comment:
- [ ] On your work section, kindly work on the margin of the elements and ensure they are uniform to match the Figma design. Kindly improve that on all your cards in the `work section`

![image](https://user-images.githubusercontent.com/98453979/198649113-aa1630a0-9458-4084-a3d6-1b542a7d58d3.png)

### Checkout Figma
![image](https://user-images.githubusercontent.com/98453979/198649646-2bad8feb-1ba8-4dea-8f76-08d08fefba9d.png)

- [ ] Kindly space these elements to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/198650452-d90b3392-9d03-4e46-8a3d-04e68710d4ab.png)

### Checkout Figma
![image](https://user-images.githubusercontent.com/98453979/198651073-4ed87764-2d96-4c45-8961-f455afce8c77.png)

- [ ] Kindly center your cards to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/198652491-72954244-b049-4aa6-882e-bfbcf6f1b224.png)

- [ ] Kindly also adjust the margins of your about section to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/198654159-3abd1d48-3206-4160-a69d-6badb56b0c9a.png)


![image](https://user-images.githubusercontent.com/98453979/198655065-1283374c-2694-4961-b8cd-08a4f7dd4f04.png)